.github/workflows/CI.yml

```
@@ -0,0 +1,117 @@
+  build_and_test:
+    name: Build and test
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout codes
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # Specify changes
+      - name: Check for changes
+        uses: dorny/paths-filter@v2
```

actionlint (1.7.8)

the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)